src/functionaltest/java/com/ericsson/ei/frontend/SubscriptionHandlingFunctionality.java

```diff
@@ -26,12 +27,16 @@
             "resources", "responses", "SubscriptionTemplate.json");
     private static final String RELOAD_TEST_FILE_PATH = String.join(File.separator, "src", "functionaltest",
             "resources", "responses", "SubscriptionForUploadCase.json");
+    private static final String RELOAD_TEST_FILE_PATH_LDAP = String.join(File.separator, "src", "functionaltest",
```

Comment:
This filename should maybe explain what it is rather than what it is meant for (e.g SUBSCRIPTION_FOR_UPLOAD_LDAP_FILEPATH or a better name), there could be other tests using this filepath aswell

Reply:
Well, it is for "reload" button so the name... Nevertheless I try to give it more meaningful name